tcms/testplans/tests/test_new_plan.py

```diff
@@ -70,4 +77,48 @@ def _test_plan_create_new(self, is_active):
         self.assertTrue(plan.emailing.auto_to_case_owner)
         self.assertTrue(plan.emailing.auto_to_case_default_tester)
         self.assertFalse(plan.emailing.notify_on_plan_update)
-        self.assertFalse(plan.emailing.notify_on_case_update)
\ No newline at end of file
+        self.assertFalse(plan.emailing.notify_on_case_update)
+
+
+class TestNewPlanWithoutPermission(BaseNewPlanTest):
+    def setUp(self):
+        super().setUp()
+
+        self.user.is_superuser = False
+        self.user.save()
+
+        remove_perm_from_user(self.user, 'testplans.add_testplan')
+
+    def test_get_no_perm(self):
```

Comment:
Test method names must reflect what the test does. They need to be longer and more descriptive.